ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -44,7 +44,9 @@ message Commands {
   // Party on whose behalf the command should be executed. It is up to the server to verify that the
   // authorisation can be granted and that the connection has been authenticated for that party.
   // Must be a valid PartyIdString (as described in ``value.proto``).
-  // Required
+  // Deprecated in favor of the ``act_as`` field. If both are set, then the effective list of parties on whose
+  // behalf the command should be executed is the union of all parties listed in ``party`` and ``act_as``.
+  // Optional
```

Comment:
TODO: adjust the text on command deduplication to explain how it works with `actAs` parties.